### DISCUSSION TITLE
we figure if its an auto generated [Sep 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3935] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Casey**  
#### ok thank you I was on 0.92.4, this seems to be working better in 0.96.4 although it interestingly comes back with a blank row error first. The row does get saved and can be seen after using the refresh button. I can live with that for now. 🙏  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1021527914220634192)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
